### PR TITLE
[vsphere] support for listing processes in guest OS

### PR DESCRIPTION
**Pull request by ireznice**

### CODE REVIEW

File: lib/fog/vsphere/requests/compute/list_processes.rb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+              exit_code: pi.exitCode,
+              name: pi.name,
+              owner: pi.owner,
+              pid: pi.pid,
+              start_time: pi.startTime
+            )
+          end
+        end
+
+        def list_processes(vm_id, opts = {})
```
Comment (plribeiro3000) [inline lib/fog/vsphere/requests/compute/list_processes.rb]: I guess you forgot to add `Class Mock` here?

Comment (ireznice) [inline lib/fog/vsphere/requests/compute/list_processes.rb]: Yes, I did, sorry about that

Comment (plribeiro3000) [inline lib/fog/vsphere/requests/compute/list_processes.rb]: Hey, no problem. =)

**Comment by plribeiro3000** [comment]
:+1:

**Comment by geemus** [comment]
Thanks!

**Pull request merged**